appium/lib/screen-object/accessors/element.rb

```diff
@@ -239,23 +231,29 @@ def get_element_by_exact_text(value)
         end
       end
 
+      def is_a_string?(argument)
```

Comment:
This is unnecessary, remove and refactor uses of this method to be `argument.is_a? String`